### DISCUSSION TITLE
setPassword: {"code":3,"message":"no such function","data":null}

### DISCUSSION THREAD

**Discussion by fraengii** [ether/etherpad-lite#5058] [Q&A] [answered]
Hi,

I'm trying to get familiar with the API and I already had some cognition (which I would like to share — is there a way to take part writing the documentation?).
But I always tumble over some command like the one above which stubbornly denied to do what I suppose him to do. I use it with curl:
```
curl "https://<myserver>/api/1.2.15/setPassword?apikey=<myAPIkey>&padID=<thePadID>&password=<myPassword>"
```
In the documentation ist written 
> **setPassword(padID, password)**

which, as I experienced, has to be interpreted like I've done above. But I get the error message which I quoted in the subject.
Where am I wrong?

Kind regards
Fraengii

**Comment by webzwo0i** [comment, marked as answer]
Thanks for offering help with our docs. We collect documentation on our [wiki](https://github.com/ether/etherpad-lite/wiki) and some is also [auto-generated](https://etherpad.org/doc/latest/). We welcome any help in making this better.

Regarding your question: We dropped support for setting specific group pad passwords as of https://github.com/ether/etherpad-lite/commit/66df0a572f03d1c9cbe7364ea8a81a3c05c17518 I assume some of our documentation is outdated if you found a hint to `setPassword`?

**Reply by fraengii** [reply]
Thanks, @webzwo0i!
I noticed that I landed on a documentation page for version 1.8.4. That was the fault.
